KOGITO-5401 KOGITO-5683 - Remove Kogito Quarkus archetype (#3832)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/chap-kogito-release-notes.adoc

quarkus.datasource.password=kogito-pass
------

==== Renamed ArtifactId of {PRODUCT} Spring Boot archetype 

> The {PRODUCT} Spring Boot archetype ArtifactId has been renamed from `kogito-springboot-archetype` to `kogito-spring-boot-archetype`.

Reviewer: ```suggestion
The name of the {PRODUCT} Spring Boot archetype ArtifactId has changed from `kogito-springboot-archetype` to `kogito-spring-boot-archetype`.
```
Removed passive voice.

> To create a new Spring Boot based project using the archetype, please revisit updated instructions on {URL_CREATING_RUNNING}#chap-kogito-creating-running[_{CREATING_RUNNING}_].

Reviewer: ```suggestion
To create a new Spring Boot based project using the archetype, see the instructions on {URL_CREATING_RUNNING}#chap-kogito-creating-running[_{CREATING_RUNNING}_].
```

==== Removed {PRODUCT} Quarkus archetype 

> The {PRODUCT} Quarkus archetype (`kogito-quarkus-archetype`) has been removed. To create a new Quarkus based project, please revisit updated instructions on {URL_CREATING_RUNNING}#chap-kogito-creating-running[_{CREATING_RUNNING}_].

Reviewer: ```suggestion
The {PRODUCT} Quarkus archetype (`kogito-quarkus-archetype`) is no longer available. To create a new Quarkus based project, please revisit updated instructions on {URL_CREATING_RUNNING}#chap-kogito-creating-running[_{CREATING_RUNNING}_].
```
Removed passive voice.
